lib/jsonwp-proxy/proxy.js

```diff
@@ -97,6 +97,7 @@ class JWProxy {
       throw new Error(`Could not find :session section for url: ${remainingUrl}`);
     }
     remainingUrl = remainingUrl.replace(/\/$/, ''); // can't have trailing slashes
+
```

Comment:
does it make sense to log the final url?

Reply:
It's logged in the "proxying x to y" line.